yt/data_objects/construction_data_containers.py

```diff
@@ -950,6 +950,28 @@ def _fill_particles(self, part):
         for p in part:
             self[p] = self._data_source[p]
 
+    def _check_sph_type(self, fi):
```

Comment:
Can we change `fi` to `finfo` for this function?

Reply:
Done